install/bootstrap_livecd_remote_iso.md

```diff
@@ -224,343 +246,280 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
-   pit:/var/www/ephemeral# cd
+   pit:/var/www/ephemeral# rpm -Uvh --force $(find ./${CSM_RELEASE}/rpm/cray/csm/ -name "cray-site-init-*.x86_64.rpm" | sort -V | tail -1)
    ```
 
 1. Show the version of CSI installed.
 
    ```bash
-   pit# csi version
+   pit# /root/bin/metalid.sh
    ```
 
    Expected output looks similar to the following:
    ```
+   = PIT Identification = COPY/CUT START =======================================
+   VERSION=1.5.7
+   TIMESTAMP=20211028194247
+   HASH=ge4aceb1
    CRAY-Site-Init build signature...
-   Build Commit   : b3ed3046a460d804eb545d21a362b3a5c7d517a3-release-shasta-1.4
-   Build Time     : 2021-02-04T21:05:32Z
-   Go Version     : go1.14.9
-   Git Version    : b3ed3046a460d804eb545d21a362b3a5c7d517a3
+   Build Commit   : a6c8dddf9df1a9fc7f8c4f17cb26568a8b41d433-main
+   Build Time     : 2021-12-01T16:16:41Z
+   Go Version     : go1.16.10
+   Git Version    : a6c8dddf9df1a9fc7f8c4f17cb26568a8b41d433
    Platform       : linux/amd64
-   App. Version   : 1.5.18
+   App. Version   : 1.12.2
+   metal-net-scripts-0.0.2-1.noarch
+   metal-basecamp-1.1.9-1.x86_64
+   metal-ipxe-2.0.10-1.noarch
+   pit-init-1.2.12-1.noarch
+   = PIT Identification = COPY/CUT END =========================================
    ```
 
-1. Download and install/upgrade the workaround and documentation RPMs.
+<a name="before-configuration-payload-workarounds"></a>
+#### 4.1 Before Configuration Payload Workarounds
 
-   If this machine does not have direct Internet access, these RPMs will need to be externally downloaded and then copied to the system.
+Follow the [workaround instructions](../update_product_stream/index.md#apply-workarounds) for the `before-configuration-payload` breakpoint.
 
-   **Important:** In an earlier step, the CSM release plus any patches, workarounds, or hotfixes
-   were downloaded to a system using the instructions in [Check for Latest Workarounds and Documentation Updates](../update_product_stream/index.md#workarounds). Use that set of RPMs rather than downloading again.
+<a name="generate-installation-files"></a>
+#### 4.2 Generate Installation Files
 
-   ```bash
-   linux# wget https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.2/noarch/docs-csm-latest.noarch.rpm
-   linux# wget https://storage.googleapis.com/csm-release-public/shasta-1.5/csm-install-workarounds/csm-install-workarounds-latest.noarch.rpm
-   linux# scp -p docs-csm-*rpm csm-install-workarounds-*rpm ncn-m001:/root
-   linux# ssh ncn-m001
-   pit# rpm -Uvh --force docs-csm-latest.noarch.rpm
-   pit# rpm -Uvh --force csm-install-workarounds-latest.noarch.rpm
-   ```
+1. **For second-time installs (re-installs) where the `system_config.yaml`  parameter file is available**,generate the updated system configuration (see [avoiding parameters](../background/cray_site_init_files.md#save-file--avoiding-parameters)).
```

Comment:
You have two spaces behind `system_config.yaml` ... no big deal, but it would generate a subsequent PR when the format checker is run against this repo.